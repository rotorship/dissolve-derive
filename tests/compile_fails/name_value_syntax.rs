use dissolve_derive::Dissolve;

#[derive(Dissolve)]
struct NameValueSyntax {
	#[dissolved = "skip"]
	field: String,
}

fn main() {}
