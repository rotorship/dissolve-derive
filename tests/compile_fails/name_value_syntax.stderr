error: dissolved attribute should use list syntax: #[dissolved(rename = "new_name")] instead of #[dissolved = ...]
 --> tests/compile_fails/name_value_syntax.rs:5:2
  |
5 |     #[dissolved = "skip"]
  |     ^^^^^^^^^^^^^^^^^^^^^
